model/lib/extensions/email_address_extension.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:jmap_dart_client/jmap/mail/email/email_address.dart';
+
+extension EmailAddressExtension on EmailAddress {
+
+  String getNameEmail() {
```

Comment:
why not `toString()`?

Reply:
Extensions can't declare members with the same name as a member declared by 'Object'.